x-pack/test/api_integration/services/usage_api.js

```diff
@@ -9,6 +9,25 @@ export function UsageAPIProvider({ getService }) {
   const supertest = getService('supertest');
   const supertestNoAuth = getService('supertestWithoutAuth');
 
+  async function getStats(test) {
```

Comment:
cc @tsullivan. hopefully it makes sense from the PR description why this change is necessary